frontends/mit-open/src/pages/OnboardingPage/OnboardingPage.tsx

```diff
@@ -112,45 +115,186 @@ const Label = styled.div({
   margin: "0 0 40px",
 })
 
+const profileSchema = yup.object().shape({
```

Comment:
`profileSchema` isn't really doing much right now. I mostly added it out of habit.

I also made it a bit loose, though with something like
```
  goals: yup
    .array()
    .of(yup.string().oneOf(GOALS_CHOICES.map((choice) => choice.value))),
```
it will validate the membership and get the TS correct, though I'm not sure there's much point for checkboxes / radio boxes. / select.